src/main/cpp/mechanisms/noteManager/decoratormods/feederIntakeState.cpp

```diff
@@ -65,9 +65,12 @@ bool feederIntakeState::AtTarget()
 	return attarget;
 }
 
-bool feederIntakeState::IsTransitionCondition ( bool considerGamepadTransitions )
+bool feederIntakeState::IsTransitionCondition(bool considerGamepadTransitions)
 {
 	// To get the current state use m_mechanism->GetCurrentState()
 
-	return ( considerGamepadTransitions && TeleopControl::GetInstance()->IsButtonPressed ( TeleopControlFunctions::EXAMPLE_MECH_FORWARD ) );
+	auto currentstate = m_mechanism->GetCurrentState();
+	bool noSensorsDetected = m_mechanism->getlauncherSensor()->Get() == false && m_mechanism->getplacerInSensor()->Get() == false && m_mechanism->getplacerMidSensor()->Get() == false && m_mechanism->getplacerOutSensor()->Get() == false && m_mechanism->getbackIntakeSensor()->Get() == false && m_mechanism->getfrontIntakeSensor()->Get() == false && m_mechanism->getfeederSensor()->Get() == false;
+
+	return ((TeleopControl::GetInstance()->IsButtonPressed(TeleopControlFunctions::INTAKE) && m_mechanism->isLauncherMode()) || (noSensorsDetected && currentstate == m_genState->GetnoteManager()->STATE_PLACER_TO_LAUNCHER_FRONT));
```

Comment:
Static cast as a Int the  state placer to launcer front

static_cast<int>(m_mechanism->STATE_)

and emter after the OR